.github/workflows/update-spellcheck-binaries.yml

```diff
@@ -0,0 +1,71 @@
+name: Update node-spellchecker binaries
+on:
+  push:
+    paths:
+      - 'lib/bin/node-spellchecker/**'
+jobs:
+  windows-binaries:
+    name: Windows Binaries
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: npm ci
```

Comment:
`ci` is `install` but stricter and making sure that it does not modify the lockfile -> for reproducible builds.